File: go/isogram/isogram.go
package isogram

import (
	"strings"
)

//IsIsogram determines whether a word or phrase is an isogram
func IsIsogram(s string) bool {
	lst := strings.Split(strings.ToLower(s), "")
	repeated := make(map[string]bool)
	for _, c := range lst {
		if c == "-" || c == " " {
			continue
		}
		if repeated[c] {

			return false
		}
		repeated[c] = true
	}
	return true
}

File: python/isogram/isogram.py
def is_isogram(string):
    # Alternatively: Counter(string.lower()).most_common(1)[0][1] == 1
    repeated = {}
    for ch in string.lower():
        if ch == "-" or ch == " ":
            continue
        if ch in repeated:
            return False
        repeated[ch] = True
    return True
